promote-release-to-box.py

```diff
@@ -0,0 +1,344 @@
+import os
+import re
+import sys
+import json
+import yaml
+import argparse
+import requests
+import logging
+import subprocess
+from tqdm import tqdm
+from boxsdk import OAuth2,Client
+from artifactory import ArtifactoryPath
+from requests.auth import HTTPBasicAuth
+from boxsdk.exception import BoxAPIException
+
+
+def get_artifact_info_json(build_name, build_number, rt_auth = (None, None), rt_base_url = None, jf_cli_rt_name = 'artifactory'):
+    ''' 
+    Expected Jfrog CLI is availble in the system.
+    Executes these:
+    1. jf config add orl-artifactory --interactive=false \
+        --enc-password=false --basic-auth-only \
+        --artifactory-url https://repo.orl.eng.hitachivantara.com/artifactory \
+        --password --user buildguy
+
+    2. jf rt search --server-id orl-artifactory \
+        --props "build.name=pdi-xxx-9.5.1.0;build.number=86" "*-9.5.1.0-86.zip" \
+        > builds.json
+
+    Alternative way: Use ArtifactoryBuildManager https://github.com/devopshq/artifactory#builds
+    
+    ^^^^^^^^^^^^^
+    from artifactory import ArtifactoryBuildManager
+
+    arti_build = ArtifactoryBuildManager(
+        orl_url, auth=auth_orl)
+
+    # Get all builds,space turns into %
+    all_builds = arti_build.builds
+    print(all_builds)
+    ^^^^^^^^^^^^^
+    '''
+    
+
+    # # Define the command and arguments
+    # command = [
+    #     'jf', 'config', 'add', f'{jf_cli_rt_name}',
+    #     '--interactive=false', '--enc-password=false', '--basic-auth-only',
+    #     '--artifactory-url', f'{rt_base_url}',
+    #     '--password', f'{rt_auth[0]}',
+    #     '--user', f'{rt_auth[1]}'
+    # ]
```

Comment:
if `jf_cli_rt_name` is null then perform the jfrog add config.

Reply:
Check added